packages/quantic/force-app/main/default/lwc/exampleSearch/exampleSearch.css

```diff
@@ -1,3 +0,0 @@
-.main.slds-grid {
```

Comment:
@nathanlb so did you find a fix so the page isn't jumping in width whenever there's no result?

Reply:
No but if it's ok with y'all I think it would be better to wait till we have all our search page components (including the NoResults and loading animation stencils, which should at least stop it from collapsing) before we attempt to fix it properly.

Reply:
I propose we wait till we have all our basic components (including No Results and loading animation stencils) which should atleast prevent it from collapsing. Then I think we can best find the way to fix it.